cmd/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -93,6 +99,7 @@ func NewCommandStartDiscoveryServer(out, err io.Writer) *cobra.Command {
 	o.Authorization.AddFlags(flags)
 	flags.StringVar(&o.ProxyClientCertFile, "proxy-client-cert-file", o.ProxyClientCertFile, "client certificate used identify the proxy to the API server")
 	flags.StringVar(&o.ProxyClientKeyFile, "proxy-client-key-file", o.ProxyClientKeyFile, "client certificate key used identify the proxy to the API server")
+	flags.StringVar(&o.ConfigFilePath, "config-file-path", o.ConfigFilePath, "path to a config file")
```

Comment:
Document this better, where should a user look to find the contents of this config file?